app/templates/_bower.json

```diff
@@ -2,8 +2,8 @@
   "name": "<%= app %>",
   "main": "<%= app %>.js",
   "version": "0.0.1",
-  "homepage": "https://github.com/<% username %>/<%= app %>",
-  "authors": ["<%= username%> <email>"],
+  "homepage": "https://<% repohosturl %>/<% usernamespace %>/<%= app %>",
```

Comment:
Missing `=` before `repohosturl` and `usernamespace` (the latter was my fault, :laughing:)